src/components/Pagination/Pagination.stories.tsx

```diff
@@ -1,20 +1,78 @@
 import * as React from "react";
 import { action } from "@storybook/addon-actions";
+import { withDesign } from "storybook-addon-designs";
```

Comment:
Is the lack of knobs intended (Eg consumers aren't ever supposed to use this component with button/label text that's different?)

Reply:
I'll add these in!